librz/core/cmd_eval.c

```diff
@@ -238,7 +238,7 @@ static void nextpal(RzCore *core, int mode) {
 						rz_list_free(files);
 						return;
 					}
-					eprintf("%s %s %s\n", nfn, curtheme, fn);
+					eprintf("%s %s %s\n", nfn ? nfn : "(null)", curtheme, fn);
```

Comment:
By the way, I think we can use `rz_str_get{2}()` in these cases

Reply:
If we want to use such thing though we should make the function static inline and move it to rz_str.h because it is so small that a call would be wasted time

Reply:
agree